src/rust/src/orchard_bundle.rs

```diff
@@ -66,29 +138,48 @@ impl Bundle {
         self.inner().map(|b| b.actions().len()).unwrap_or(0)
     }
 
+    /// Returns whether the Orchard bundle is present and spends are enabled.
     pub(crate) fn enable_spends(&self) -> bool {
         self.inner()
             .map(|b| b.flags().spends_enabled())
             .unwrap_or(false)
     }
 
+    /// Returns whether the Orchard bundle is present and outputs are enabled.
```

Comment:
Documentation is wrong.

Reply:
(This is for `enable_outputs`)

Reply:
Likewise, this _is_ documenting `enable_outputs` - which is clearer here because you can see `enable_spends` above this documentation.